test/main.js

```diff
@@ -133,6 +170,37 @@ const createMocha = (driver, testCase) => {
   // By default `require` caches files, making it impossible to require the same file multiple times.
   // Since we want to execute the same tests against many browsers we need to prevent this behaviour by
   // clearing the require cache.
+  mocha.suite.beforeAll('Do something before everything', function () {
+    global.isRemoteBrowser = isRemoteBrowser
+    global.browserName = browserName
+  })
+  mocha.suite.beforeEach('Set retry', function () {
+    this.currentTest.retries(2)
+  })
+  mocha.suite.afterEach('Capture total number of test failures', function () {
+    const currentTestState = this.currentTest.state
+    //console.log(`The current test state is: ${currentTestState}`)
```

Comment:
You might have forgot this log here

Reply:
That was actually just for my debugging, its not needed.